compiler/noirc_frontend/src/parser/mod.rs

```diff
@@ -236,6 +235,55 @@ pub struct ParsedModule {
     pub submodules: Vec<SubModule>,
 }
 
+/// A ParsedModule contains an entire Ast for one file.
+#[derive(Clone, Debug, Default)]
+pub struct ParsedModule {
+    pub items: Vec<Item>,
+}
+
+impl ParsedModule {
+    pub fn into_unorder(self) -> UnorderParsedModule {
```

Comment:
Ok, now I see it means it is the separated kind. Perhaps rename Unorder[ed] to `SortedModule` then.